concepts/05 UI Components/DataGrid/30 Filtering and Searching/6 API/5 Filtering Columns Bound to Collection/Filtering Columns Bound to Collection.md

```diff
@@ -0,0 +1 @@
+This section describes how to bind a DataGrid column to the data that contains a collection, and implement filtering by this column both on the client-side and server-side. 
```

Comment:
> **the** data that contains a collection

This case is not definite because **some** data contains **some** collection. Therefore, the case is indefinite.

> on **the** client-side and server-side

1. Incorrect article usage. We do not use a definite article when we iterate things with AND. Refer to the latest guideline on articles.
2. As I mentioned before, a hyphen indicates an adjective: **end-user** functionality, **client-side** implementation, etc. In this particular case, we use client side and server side as simple nouns.
3. We repeat the word "side" here. We definitely can "move it outside the brackets" and use it only once.

That's my current take on this sentence:

```suggestion
This section describes how to bind a DataGrid column to data that contains a collection, and implement filtering by this column both on client and server side. 
```